projects/demo/src/modules/components/calendar/examples/5/index.ts

```diff
@@ -12,11 +12,18 @@ const handler: TuiHandler<TuiDay, string> = (day: TuiDay) => {
 };
 
 @Component({
-    selector: 'tui-calendar-example-5',
+    standalone: true,
+    imports: [TuiCalendarModule],
     templateUrl: './index.html',
-    styleUrls: ['./index.less'],
+    styles: [
+        `
+            tui-primitive-calendar [data-type='holiday'] {
+                background-color: var(--tui-support-09);
+            }
+        `,
+    ],
```

Comment:
in this case, the backlight is inconvenient and the import of css mixins does not work normally

it is better not to move away from uniformity when a separate LESS tab is displayed for examples with styles: let's return it to a separate file, and for examples without styles, add empty index.less files

Reply:
Well, it's just one line. Maybe it's ok.